Add a brief explanation how to collect different telemetry (#1308)
grafana/alloy: docs/sources/collect/choose-component.md

---
canonical: https://grafana.com/docs/alloy/latest/collect/choose-component/
description: Find out which components are useful for which tasks

> title: Choose an Alloy component

Reviewer: ```suggestion
title: Choose a Grafana Alloy component
menuTitle: Choose a component
```

weight: 100
---

> # Choose an Alloy component
> 
> [Components][components] are the building blocks of Alloy, and there is a [large number of them][components-ref].

Reviewer: ```suggestion
# Choose a  {{< param "FULL_PRODUCT_NAME" >}} component

[Components][components] are the building blocks of {{< param "FULL_PRODUCT_NAME" >}}, and there is a [large number of them][components-ref].
```

We can use a shorter menu title, and then use the full product name in the title and H1.

## Logs from infrastructure

Use `loki.*` components to collect infrastructure logs.
The `loki.*` components label your logs in a way that resembles Prometheus metrics.
This makes it easy to correlate infrastructure metrics collected by `prometheus.*` components
with logs collected by `loki.*` components.

For example, the label that both `prometheus.*` and `loki.*` components would use for a Kubernetes namespace is called `namespace`.
On the other hand, gathering logs using an `otelcol.*` component might use the [OpenTelemetry semantics][OTel-semantics] label called `k8s.namespace.name`,
which wouldn't correspond to the `namespace` label that is common in the Prometheus ecosystem.

## Logs from applications

Use `otelcol.receiver.*` components to collect application logs.
This will gather the application logs in an OpenTelemetry-native way, making it easier to 
correlate the logs with OpenTelemetry metrics and traces coming from the application.
All application telemetry must follow the [OpenTelemetry semantic conventions][OTel-semantics], simplifying this correlation.

> For example, if your application runs on Kubernetes, every trace, log, and metric can have a `k8s.namespace.name` resource attribute.
> 
> 
> [OTel-semantics]:https://opentelemetry.io/docs/concepts/semantic-conventions/

Reviewer: ```suggestion
For example, if your application runs on Kubernetes, every trace, log, and metric can have a `k8s.namespace.name` resource attribute.

[OTel-semantics]:https://opentelemetry.io/docs/concepts/semantic-conventions/
```

Removing extra CR/LF
